Source/AliveLib/Scrab.cpp

```diff
@@ -692,22 +693,38 @@ void Scrab::M_RunJumpEnd_13_4A9BE0()
 
 void Scrab::M_WalkToFall_14_4A9460()
 {
-    NOT_IMPLEMENTED();
+    M_JumpToFall_8_4A9220();
+    if (field_106_current_motion == 0)

```

Comment:
For any comparison with field_106_current_motion against a number replace with the motion enum name